v03_pipeline/lib/tasks/reference_data/updated_reference_dataset_collection_test.py

```diff
@@ -343,3 +373,70 @@ def test_update_task_with_existing_reference_dataset_collection_table(
                 ),
             ],
         )
+
+    @mock.patch.dict(
+        'v03_pipeline.lib.reference_data.compare_globals.CONFIG',
+        MOCK_CONFIG,
+    )
+    @mock.patch.dict(
+        'v03_pipeline.lib.reference_data.dataset_table_operations.CONFIG',
+        MOCK_CONFIG,
+    )
+    @mock.patch(
+        'v03_pipeline.lib.tasks.reference_data.updated_reference_dataset_collection.ValidateCallsetTask',
+    )
+    @mock.patch.object(ReferenceDatasetCollection, 'datasets')
+    def test_update_mito_rdc_additional_field(
+        self,
+        mock_rdc_datasets,
+        mock_validate_callset_task,
+    ) -> None:
+        mock_validate_callset_task.return_value = MockCompleteTask()
+        mock_rdc_datasets.return_value = ['mitimpact']
+
+        # Reference dataset collection has older mitimpact version, path, selects
+        shutil.copytree(
+            TEST_COMBINED_MITO_1,
+            valid_reference_dataset_collection_path(
+                ReferenceGenome.GRCh38,
+                DatasetType.MITO,
+                ReferenceDatasetCollection.COMBINED,
+            ),
+        )
+
+        worker = luigi.worker.Worker()
+        task = UpdatedReferenceDatasetCollectionTask(
+            reference_genome=ReferenceGenome.GRCh38,
+            dataset_type=DatasetType.MITO,
+            reference_dataset_collection=ReferenceDatasetCollection.COMBINED,
+            sample_type=SampleType.WGS,
+            callset_path=TEST_SNV_INDEL_VCF,
+            project_guids=[],
+            project_remap_paths=[],
+            project_pedigree_paths=[],
+            skip_validation=True,
+            run_id='2',
+        )
+        worker.add(task)
+        worker.run()
+        self.assertTrue(task.complete())
+
+        # Expect additional mitimpact select field to be present in rdc
```

Comment:
Did we really not have coverage for an additional field?  Or was it only in `compare_globals_tests`?

More coverage is good though!

Reply:
true it's in `test_get_datasets_to_update_select_different` lol

Reply:
I removed this test